ethers-solc/src/cache.rs

```diff
@@ -25,11 +26,34 @@ use std::{
 /// `ethers-solc` uses a different format version id, but the actual format is consistent with
 /// hardhat This allows ethers-solc to detect if the cache file was written by hardhat or
 /// `ethers-solc`
-const ETHERS_FORMAT_VERSION: &str = "ethers-rs-sol-cache-3";
+const ETHERS_FORMAT_VERSION: &str = "ethers-rs-sol-cache-4";
 
 /// The file name of the default cache file
 pub const SOLIDITY_FILES_CACHE_FILENAME: &str = "solidity-files-cache.json";
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Hash)]
+pub struct CompilationUnitId(String);
```

Comment:
should this take the hasher outcome u64 instead of String and print hex via Display impl?

Reply:
https://github.com/gakonst/ethers-rs/pull/1964/commits/4439d1435cf44b8cb76d7d82ebc52a051ee99e47